src/App.tsx

```diff
@@ -16,80 +16,151 @@ export const goodsFromServer = [
 ];
 
 enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
+  LENGTH = 'length',
+  ALPHABETICAL = 'alphabetical',
+  NONE = '',
 }
 
-type ReorderOptions = {
+interface ReorderOptions {
   sortType: SortType,
   isReversed: boolean,
-};
+}
 
-// Use this function in the render method to prepare goods
-export function getReorderedGoods(
-  goods: string[],
-  { sortType, isReversed }: ReorderOptions,
-) {
+export function getReorderedGoods(goods: string[],
+  { sortType, isReversed }: ReorderOptions) {
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case SortType.ALPHABETICAL:
+      visibleGoods.sort((a: string, b: string) => {
+        if (a < b) {
+          return -1;
+        }
+
+        if (a > b) {
+          return 1;
+        }
+
+        return 0;
+      });
+      break;
+
+    case SortType.LENGTH:
+      visibleGoods.sort((a: string, b: string) => {
+        if (a.length < b.length) {
+          return -1;
+        }
+
+        if (a.length > b.length) {
+          return 1;
+        }
+
+        return 0;
```

Comment:
Alwyas simplify your code.

```suggestion
        return a.length - b.length
```